pypeit/calibrations.py

```diff
@@ -944,7 +941,7 @@ def default_steps():
 
         """
         # Order matters!
-        return ['bpm', 'bias', 'arc', 'tiltimg', 'slits', 'wv_calib', 'tilts', 'flats', 'wave']
+        return ['bias', 'bpm', 'arc', 'tiltimg', 'slits', 'wv_calib', 'tilts', 'flats', 'wave']
```

Comment:
This should now be an if/then statement where you swap only 
if `par['makebpm'] = True`.  Make sense?

Reply:
Or maybe you are right that we can always handle this ordering..

Reply:
Since the BPM is never used for the construction of the bias frame, I think this new order is OK, perhaps preferred, if the user wants to generate their own  BPM.

Reply:
yes, I now agree.